src/main/java/dev/blachut/svelte/lang/parsing/html/SvelteHtmlParsing.kt

```diff
@@ -1,6 +1,6 @@
 package dev.blachut.svelte.lang.parsing.html
 
-import com.intellij.codeInsight.daemon.XmlErrorMessages
+import com.intellij.codeInsight.daemon.XmlErrorBundle
```

Comment:
Unfortunately this change breaks 2019.2 and I guess also 2019.3, so I can't merge this yet.

![image](https://user-images.githubusercontent.com/615013/88668535-0a77d300-d0e3-11ea-90a5-5d4bf029ec19.png)